### PR TITLE
Add Boosty icon

### LINKED ISSUE #8894: Request: Boosty Icon
### Brand Name

Boosty

### Website

https://boosty.to/

### Popularity Metric

The Similarweb rank is `4,523`.
See https://www.similarweb.com/website/boosty.to

### Official Resources for Icon and Color

Example:
svg: https://static.boosty.to/assets/images/boostyLogo.60ihp.svg ([archive](https://static.boosty.to/static/boosty_logo_svg.zip))
color: `#F15F2C`

From the official [logo and corporate identity](https://boosty.to/app/brand) page.

### Additional Comments

There are two variants of this icon:
* simple logo: https://static.boosty.to/assets/images/boostyLogo.60ihp.svg
* logo with domain: https://static.boosty.to/assets/images/boostyDomainLogo.5Vlxt.svg

### Contributing

Yes

### PR DESCRIPTION
**Pull request by Gautam25Raj**
boosty

**Issue:** closes #8894

**Similarweb rank:**
The Similarweb rank is 4,523. See https://www.similarweb.com/website/boosty.to

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from boosty website which was provided as SVG https://boosty.to/app/brand?locale=en_US
Color: `#F15F2C` Mentioned in the boosty website.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1720,6 +1720,12 @@
             "hex": "F7901E",
             "source": "https://www.boostmobile.com"
         },
+        {
+            "title": "Boosty",
+            "hex": "F15F2C",
+            "source": "https://boosty.to/app/brand?locale=en_US",
+            "guidelines": "https://boosty.to/app/brand?locale=en_US#:~:text=PNG-,Logo%20use%20rules,-Failure%20to%20comply"
+        },
```
Comment (LitoMore) [inline _data/simple-icons.json:1728]:
```suggestion
        {
            "title": "Boosty",
            "hex": "F15F2C",
            "source": "https://boosty.to/app/brand",
            "guidelines": "https://boosty.to/app/brand"
        },
```

Those queries need to be removed.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json [hunk 2]
```diff
@@ -1720,6 +1720,12 @@
             "hex": "F7901E",
             "source": "https://www.boostmobile.com"
         },
+        {
+            "title": "Boosty",
+            "hex": "F15F2C",
+            "source": "https://boosty.to/app/brand?locale=en_US",
+            "guidelines": "https://boosty.to/app/brand?locale=en_US"
```
Comment (adamrusted) [inline _data/simple-icons.json:1727]:
```suggestion
            "source": "https://boosty.to/app/brand",
            "guidelines": "https://boosty.to/app/brand"
```

File: icons/boosty.svg
Comment (adamrusted) [inline icons/boosty.svg:1]: image

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the contribution @Gautam25Raj!
There's a couple tweaks to this that'll need making before we can accept it. I've removed the locale string on the URLs, and there appears to be a difference in the SVG source (red) and your PR (black). Probably just down to the precision of SVGO, would be worth trying a precision of 4 instead of 3!

File: icons/boosty.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Boosty</title><path d="M2.638 14.336 6.788 0h6.377L11.88 4.444l-.039.077-3.386 11.733h3.157a1461.06 1461.06 0 0 0-3.093 7.733c-5.83-.063-7.46-4.228-6.034-9.155M8.545 24l7.688-11.035h-3.258l2.838-7.073c4.863.508 7.154 4.33 5.805 8.952C20.179 19.81 14.349 24 8.672 24h-.127Z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/boosty.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Boosty</title><path d="M2.661 14.337 6.801 0h6.362L11.88 4.444l-.038.077-3.378 11.733h3.15c-1.321 3.289-2.35 5.867-3.086 7.733-5.816-.063-7.442-4.228-6.02-9.155M8.554 24l7.67-11.035h-3.25l2.83-7.073c4.852.508 7.137 4.33 5.791 8.952C20.16 19.81 14.344 24 8.68 24h-.127z"/></svg>
```

**Review by adamrusted — APPROVED** [review]

**Pull request merged**